# gnss
origin: [37.4480114, 126.6534648, 31.627] # 분수대 필테 중앙
# origin: [35.06965949, 128.57880311, 50.921] # 창원

# 수정 필요
autonomous_goal: [35.0693232, 128.5788879, 49.451] # 0 8/7일에 땄던 좌표 35.0693838, 128.5789538

# latitude, longitude, altitude


#For using dummy gps
waypoints: [
[37.4480114, 126.6534648, 31.627],
# [37.4480707, 126.6535382, 31.477],
[37.4481465, 126.6535553, 30.505], # WP4
# [37.4548096, 126.6534598, 31.122],
[37.4480114, 126.6534648, 31.627],
[37.4481465, 126.6535553, 30.505] # WP4

#  [37.4506369, 126.6527661, 39.939], # WP1
#  [37.4506328, 126.6527598, 40.039], # WP2
#  [37.448096, 126.653459, 31.122], # WP3
#  [37.448011, 126.653464, 31.627], WP4
]

goal_range: 2 #[m] # float이면 터짐 수정 필요

# Thruster
base_thrust: 1600
kp_thruster : 3.0
kd_thruster : 0.1
thrust_range: [1100, 1900]
delta_dot_max: 4 # [rad/s] 2.09

angle_num: 10
yaw_range: 30
range: 3
margin: 0.2

# image sensing
video_num: 1
target_shape: 'Rectangle'
color_to_detect: 'red'

######################################
#     지구상의 모든상황의 변수를 통제       #
######################################
front_distance_1: 2.6
yaw_target1: 160
front_distance_2: 1.5
yaw_target2: 100
yaw_target3: 290
min_safe_distance: 1.0
min_angle_diff: 15

######################################
#             PredockCal             #
######################################

front_angle_range: 
- [175, 180]
- [-185, -180]
right_angle_range: 
  - [85, 95]  # ✅ 이렇게 하면 `[[85, 95]]` 형태로 로드됨